lib/cf_obs_binary_builder/templates/libgdiplus.spec.erb

```diff
@@ -0,0 +1,79 @@
+#
+# spec file for SUSE CAP buildpack dependency
+#
+# Copyright (c) 2019 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+
+%define soname 0
+Name:           libgdiplus-<%= version %>
+Version:        <%= version %>
+Release:        1
+Summary:        This is a dependency used in SUSE CAP buildpacks
+License:        (LGPL-2.1+ or MPL-1.1) and MIT
+Group:          Cloud/Platform/scf
+Url:            https://github.com/mono/libgdiplus
+Source:         <%= source %>
+
+# Fixes 64-bit portability issue error
+Patch:          01_pngcodec.patch
+
+BuildRequires:  giflib-devel
+BuildRequires:  libjpeg-devel
+BuildRequires:  libtool
+BuildRequires:  pkg-config
+BuildRequires:  pkgconfig(cairo)
+BuildRequires:  pkgconfig(fontconfig)
+BuildRequires:  pkgconfig(freetype2)
+BuildRequires:  pkgconfig(glib-2.0)
+BuildRequires:  pkgconfig(libexif)
+BuildRequires:  pkgconfig(libpng)
+BuildRequires:  pkgconfig(libtiff-4)
+BuildRequires:  pkgconfig(x11)
+BuildRequires:  pkgconfig(xrender)
```

Comment:
The list seems quite long, non of these packages are part of the stack meta package?
https://build.opensuse.org/package/view_file/Cloud:Platform:Stack-SLE:packages/aaa_stack_build_requires/aaa_stack_build_requires.spec?expand=1